src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --colorForLinks: #00acdc;
```

Comment:
move CSS variable to `:root` selector

Reply:
Do you mean to move to html instead of body?